packages/components/atoms/f-button/stories/IconButton.stories.js

```diff
@@ -1,88 +1,144 @@
-import {
-    withKnobs,
-    boolean,
-    select,
-    text
-} from '@storybook/addon-knobs';
 import { withA11y } from '@storybook/addon-a11y';
 import {
-    CrossIcon,
-    PlusIcon,
-    MoreVerticalIcon
+    CrossIcon, PlusIcon, MoreVerticalIcon
 } from '@justeat/f-vue-icons';
+import SharedButtonArgTypes from './sharedButtonArgTypes';
 import FButton from '../src/components/Button.vue';
-import { VALID_BUTTON_TYPES, VALID_BUTTON_SIZES } from '../src/constants';
 
 export default {
     title: 'Components/Atoms/f-button',
-    decorators: [withKnobs, withA11y]
+    decorators: [withA11y],
+    parameters: {
+        backgrounds: {
+            values: [
+                { name: 'White', value: '#fff' },
+                { name: 'Dark', value: '#333' },
+                { name: 'Blue', value: '#bfe6ff' }
```

Comment:
I would probably make one of the options one of the background colours we use for components/containers in PIE.

So atm, we have:

```scss
$color-background-default: $color-grey-10;
$color-background-dark: $color-grey-70;
$color-container-default: $color-white;
$color-container-subtle: $color-grey-20;
$color-container-strong: $color-grey-30;
$color-container-dark: $color-grey-60;
```

And the greys for those map to:

```scss
$color-grey-10: #fcfcfb;
$color-grey-20: #f5f3f1;
$color-grey-30: #efedea;
$color-grey-60: #2b3639;
$color-grey-70: #242e30;
```

So I'd probably choose one of those, potentially instead of the Dark option if you wanted to keep the blue one.

Reply:
Ah thanks Ash, that's handy - I've gone for the middle one, seems to be the best for showing the light and dark icons. Added a screenshot above, let me know what you think 👍